server/models/household.js

```diff
@@ -1,9 +1,12 @@
 let mongoose = require('mongoose');
-let Student = require('./student').schema;
+
+let Student = require('./student')
+
+
 let Schema = mongoose.Schema;
 
 let HouseholdSchema = new Schema({
-    students: [Student],
+    students: [Object],
```

Comment:
this change doesn't break anything?

Reply:
I believe it shouldn't

Reply:
Ok cool. did you check if it works on staging? The PR looks good to me otherwise.

Reply:
Yeah I'm doing that right now

Reply:
Sweet! Just check that it works, you might have to change the links to see the staging ones, so anything that's "bright-futures-academy-dev.herokuapp.com" --> "bright-futures-academy-staging.herokuapp.com". You can push this to a different branch and then deploy to staging so you don't mess up this PR

Reply:
This change breaks something

Reply:
This is a pretty big PR, I think we should definitely go through it in person just so we can make sure we aren't breaking prod

Reply:
Definitely, we should definitely check this out on Tuesday.

Reply:
I believe that rebasing changed a few things, but right now it works on staging and has passed the tests.